pkg/helpers/describe/describer.go

```diff
@@ -543,6 +545,33 @@ func describeBuildTriggers(triggers []buildv1.BuildTriggerPolicy, name, namespac
 	}
 }
 
+// describeBuildVolumes returns the description of a slice of build volumes
+func describeBuildVolumes(w *tabwriter.Writer, volumes []buildv1.BuildVolume) {
+	if len(volumes) == 0 {
+		formatString(w, "Volumes", "<none>")
+		return
+	}
+	formatString(w, "Volumes", " ")
+	fmt.Fprint(w, "\tName\tSource Type\tSource\tMounts\n")
+	for _, v := range volumes {
+		var sourceName string
+		switch v.Source.Type {
+		case buildv1.BuildVolumeSourceTypeSecret:
+			sourceName = v.Source.Secret.SecretName
+		case buildv1.BuildVolumeSourceTypeConfigMap:
+			sourceName = v.Source.ConfigMap.Name
+		default:
+			sourceName = ""
```

Comment:
Make this an obvious failure, not a missing information.  Something like `Unhandled Source Type`, so people can find and fix it.  This way it will be easy to keep up to date when it slips and people notice.

Reply:
Updated in current push, also added to tests